test/golden/routes_golden_test.dart

```diff
@@ -198,6 +199,14 @@ void main() {
   });
 
   group('player_route', () {
+    /// Open the player queue screen.
+    Future<void> openPlayerQueue(WidgetTester tester) async {
```

Comment:
We already have similar functions in `player_route_test.dart`. Please place them into `test.dart` and re-use here.

Reply:
💯 Good idea. Though if we add any more of these helpers I would vote to put them in a separate file, like a `test_navigator.dart`, because `test.dart` is getting kind of crowded.

Reply:
Yeah, sounds like a good idea